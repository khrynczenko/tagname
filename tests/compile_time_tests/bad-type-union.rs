use tagname::TagName;

#[derive(TagName)]
pub union ArbitraryUnion {
    f1: u32,
    f2: f32,
}

fn main() {}
