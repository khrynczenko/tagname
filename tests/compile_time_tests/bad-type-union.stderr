error: cannot derive `TagName` for union types
 --> tests/compile_time_tests/bad-type-union.rs:4:5
  |
4 | pub union ArbitraryUnion {
  |     ^^^^^
